mlir/test/Gradient/BufferizationTest.mlir

```diff
@@ -27,3 +27,27 @@ func.func @adjoint(%arg0: f64, %arg1: index) {
     %grad = gradient.adjoint @circuit(%arg0) size(%arg1) : (f64) -> tensor<?xf64>
     return
 }
+
+///
```

Comment:
This style of comment lets the opt tool known about separate code units in a file, and `--split-input-file` flag then actually splits the input accordingly.
```suggestion
// -----
```